### DISCUSSION TITLE
Notification bar

### DISCUSSION THREAD

**Discussion by fsan2val** [AzuraCast/AzuraCast#7046] [General]
Hello

Something very basic at best.

How to hide the ![Tablero-AzuraCast](https://github.com/AzuraCast/AzuraCast/assets/69484466/198f8b48-cdf9-48a3-905f-bfc116eeff28) forever especially for users: AzuraCast is free and open source software.

thank you

**Comment by gAlleb** [comment]
Why do you need to hide that? o_0

Oh, man...

**Reply by fsan2val** [reply]
It is a bad user experience, with the other details it would already occupy almost 50% of the screen, it is bad practice, rather the radios should be in a carousel at the top or in a grid for easy access.

If you can collaborate, welcome.

**Comment by BusterNeece** [comment]
This isn't the first time we've had this request. We don't offer support in assisting users who want to remove our simple message reminding you that we're free and open-source software and that we depend on donations.

Besides, that notice only shows up to site administrators and only once every few visits, to avoid bothering users.

**Reply by fsan2val** [reply]
Thank you for not providing assistance, I hope someone else from the community can collaborate.

In case you didn't know, not only administrators can see the notification bar, users see it too, that means it doesn't work as you indicate.

**Reply by BusterNeece** [reply]
> Thank you for not providing assistance

Huh, so you not only want to remove the message in our own application that we give away for free asking for donations, but now you want to be directly disrespectful to the creators.

I hope you're able to find support, but you won't be finding it here.

**Comment by Moonbase59** [comment]
Looks like another of these "grab, run and make money of it" cases to me… sorry to say. I _could_ surely "help removing it", but _why should I_? No, sir.
